src/components/Data/Documents/DocumentBoxItem.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="DocumentBoxItem card">
+    <div class="card-content">
+      <div class="DocumentBoxItem-title">
+        <span class="card-title">{{document.id}}</span>
+        <dropdown
+          class="DocumentBoxItem-actions"
+          :id="`document-dropdown-${document.id}`"
+          >
+          <li>
+            <a
+              v-bind:class="{'disabled': !canEdit}"
+              @click="editDocument"
+              >
+              Edit
+            </a>
+          </li>
+          <li class="divider"></li>
+          <li>
+            <a
+              v-bind:class="{'disabled': !canDelete}"
+              @click="deleteDocument"
+              >
+              Delete
+            </a>
+          </li>
+        </dropdown>
+      </div>
+      <p v-for="(value, key) in documentContent" :key="value"><strong>{{key}}</strong>: {{value}}</p>
```

Comment:
The `key` used is the document content, maybe you can use the document id instead ? It's cause an error that prevent the component to render: `Avoid using non-primitive value as key, use string/number value instead.`

Reply:
And why are you not using the json formatter directive ?
You display does well for simple documents but it feel weird with nested documents.